examples/models/thyme.sh

```
@@ -0,0 +1,11 @@
+# Run and reproduce Thyme results!
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)